### PR TITLE
Added missing moveit hand config

### PR DESCRIPTION
**Pull request by ugocupcic**
A generic moveit config for our hand, developed by @guihomework

### CODE REVIEW

**Comment by guihomework** [comment]
@ugocupcic, just to make sure things are clear for you, the code you included is not the "generic" one but is usable for hand movement planning. @VahidAminZ started to work on the generic one based on that https://github.com/ubi-agni/sr-contrib/tree/F_generic_srdf_indigo

There is still a problem for executing movements of single fingers on the real hand, because there is no controller for finger groups. With fake controllers there are more controllers instantiated and it works nice.

**Comment by ugocupcic** [comment]
@guihomework yes thanks for the clarification. I meant the base of that generic config (as you said we're still working on making the generic config work).

File: sr_moveit_hand_config/.setup_assistant [hunk 1]
```diff
@@ -0,0 +1,8 @@
+moveit_setup_assistant_config:
+  URDF:
+    package: sr_description
+    relative_path: robots/shadowhand_motor.urdf.xacro
+  SRDF:
+    relative_path: config/shadowhand_motor.srdf
+  CONFIG:
+    generated_timestamp: 1405180525
```
Comment (VahidAminZ) [inline sr_moveit_hand_config/.setup_assistant]: new line at the end

Comment (ugocupcic) [inline sr_moveit_hand_config/.setup_assistant]: fixed

File: sr_moveit_hand_config/config/fake_controllers.yaml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      - LFJ5
+      - LFJ4
+      - LFJ3
+      - LFJ2
+      - LFJ1
+      - THJ5
+      - THJ4
+      - THJ3
+      - THJ2
+      - THJ1
```
Comment (VahidAminZ) [inline sr_moveit_hand_config/config/fake_controllers.yaml]: new line

File: sr_moveit_hand_config/config/ompl_planning.yaml [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    - BKPIECEkConfigDefault
+    - KPIECEkConfigDefault
+    - RRTkConfigDefault
+    - RRTConnectkConfigDefault
+    - RRTstarkConfigDefault
+    - TRRTkConfigDefault
+    - PRMkConfigDefault
+    - PRMstarkConfigDefault
+  projection_evaluator: joints(LFJ5,LFJ4)
+  longest_valid_segment_fraction: 0.05
```
Comment (VahidAminZ) [inline sr_moveit_hand_config/config/ompl_planning.yaml]: new line

**Pull request merged**